client/src/store/actions/loginPage/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { Action } from "../../actions";
+import type { TypeErrorLogin } from "../../../../app/resolvers/auth";
+
+export const FETCH_LOGIN = "loginPage/FETCH_LOGIN";
```

Comment:
Обсудили голосом, но на всякий случай еще допишу.

Экшены лучше привязывать не к страницам, а к фичам. То есть тут можно использовать нейминг

```suggestion
export const FETCH_LOGIN = "login/FETCH_LOGIN";
```

Reply:
исправил